src/screens/login/login-validation.tsx

```diff
@@ -0,0 +1,33 @@
+export const notEmpty = (email: string, password: string) => {
```

Comment:
Funções que retornam booleans geralmente a gente nomeia como se fosse uma pergunta de sim ou não. Ex: `ìsNotEmpty`.

Reply:
Mas outra reflexão a respeito dessa nomenclatura: geralmente é mais claro de entender, pra quem está lendo, se a pergunta for para o caso afirmativo, ou seja, um `isEmpty`.

Reply:
Os comments valem pras outras funções desse arquivo tbm.